contrib/go-chi/chi/option.go

```diff
@@ -71,3 +74,15 @@ func WithAnalyticsRate(rate float64) Option {
 		}
 	}
 }
+
+// WithIsErrorCheck sets the function which is used to decide whther the
+// status should be marked as an error
+func WithIsErrorCheck(isError func(statusCode int) bool) Option {
+	return func(cfg *config) {
+		cfg.isError = isError
+	}
+}
+
+func defaultIsError(statusCode int) bool {
```

Comment:
It would probably be clearer to call this `isServerError` to indicate it checks for `5xx` error codes.